README.md

```diff
@@ -1 +1,175 @@
-# aqi-calculator
\ No newline at end of file
+# Air Quality Index (AQI) Calculator
+
+> Free and open AQI calculator. 
```

Comment:
We don't need to `free and open`
Also mention the `excel` link that we used

Reply:
Also fix the lint fixes in Markdown

Reply:
Also this calculates as per Indian AQI. US and china are different so mention tha